docs/standard/index.md

```diff
@@ -14,63 +14,51 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 
 # .NET Platform Guide
 
-> [!NOTE]
-> Check out the ["Getting Started with .NET Core" tutorials](../core/getting-started.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running.
+The .NET Platform Guide provides a large amount of information about the .NET Platform.  Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other sections of the .NET documentation.
 
-.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
+## New to .NET
 
-C#, F# and Visual Basic are popular languages that target and rely on the .NET platform. The .NET languages are known for key features such as their asynchronous programming model, language-integrated query, generic types and type system reflection. The languages also provide great options for both object-oriented and functional programming paradigms.
+If you're new to .NET, check out the [Getting Started](getting-started.md) article.
```

Comment:
We've been given guidance that the gerund forms of verbs are bad for SEO. We've been updating the 'getting started' with 'get started' based on that.  /cc @mairaw